client-misk/src/main/kotlin/app/cash/backfila/client/misk/internal/EmbeddedBackfila.kt

```diff
@@ -71,7 +71,7 @@ internal class EmbeddedBackfila @Inject internal constructor(
   override fun checkBackfillStatus(request: CheckBackfillStatusRequest): Call<CheckBackfillStatusResponse> {
     checkNotNull(request.backfill_run_id)
     // TODO(hdou): actually check the status
-    return Calls.response(CheckBackfillStatusResponse(CheckBackfillStatusResponse.Status.RUNNING))
+    return Calls.response(CheckBackfillStatusResponse(CheckBackfillStatusResponse.Status.COMPLETE))
```

Comment:
For testing purposes I'm defaulting to complete for now. Busy with other stuff so haven't been able to get back to this and do it properly